### PR TITLE
don't spam log when a zone is invisible

### PR DESCRIPTION
**Pull request by sethalves**
- don't print "not visible" repeatedly when a zone is flagged as not visible

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- create a zone, move into it
- use edit.js to uncheck the "visible" property of the zone
- note that the log isn't filled with "not visible"

**Comment by hifi-gustavo** [comment]
Build console output for commit: d4ed9993b228b59f6d7627737a8a13060c8d798a available at: https://gist.github.com/5ad942e1a045c5125ea6<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7404.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7404.dmg

File: libraries/entities-renderer/src/EntityTreeRenderer.cpp [hunk 1]
```diff
@@ -182,7 +182,9 @@ bool EntityTreeRenderer::findBestZoneAndMaybeContainingEntities(const glm::vec3&
                 // if this entity is a zone, use this time to determine the bestZone
                 if (entity->getType() == EntityTypes::Zone) {
                     if (!entity->getVisible()) {
+                        #ifdef WANT_DEBUG
                         qCDebug(entitiesrenderer) << "not visible";
+                        #endif
```
Comment (ZappoMan) [inline libraries/entities-renderer/src/EntityTreeRenderer.cpp]: As long as you're in there, can you fix the `else {` clause below to follow the coding standard?

**Comment by hifi-gustavo** [comment]
Build console output for commit: f8b8877369916e0c6c9900501bec544d27852b8c available at: https://gist.github.com/5ad942e1a045c5125ea6<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7404.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7404.dmg

**Pull request merged**